### ISSUE TITLE
Don't strip licenses from vendor packages

### ISSUE THREAD

**Issue by GDmac** [carlosbuenosvinos/compify#2]
Stripping LICENSE files is not a nice thing to do.

**Comment by jarodium** [comment]
I agree on this. You could provide an option to explicitly remove the LICENSE files. That way you leave it up to the developer to clean them.

**Comment by ebeyrent** [comment]
I have a concern for license compliance, and also want to leave the LICENSE files intact.
